### PR TITLE
feat: Adopt Typescript

### PR DESCRIPTION
**Pull request by mdovhopo**

### CODE REVIEW

**Review by sasjo — CHANGES_REQUESTED** [review]
Please keep the original TRIAL blip and instead add your new adopt ring too. This way we have history for the blip.
```
blip:
   - date: 2019-03-29
     ring: TRIAL
   - date: 2012-04-14
     ring: ADOPT
```

File: radar/dev/typescript.yaml [hunk 1]
```diff
@@ -1,16 +1,26 @@
 name: TypeScript
 logo: https://raw.githubusercontent.com/remojansen/logo.ts/master/ts.png
 blip:
-  - date: 2019-03-29
-    ring: TRIAL
+  - date: 2012-04-14
```
Comment (sasjo) [inline radar/dev/typescript.yaml:4]: Oh and fix the year to be 2021 😄

Comment (mdovhopo) [reply radar/dev/typescript.yaml:4]: oh yeah... I guess i am living in a past 😂

**Review by sasjo — APPROVED** [review]
LGTM

**Review by l0kal — APPROVED** [review]

**Pull request merged**